R/get-osmdata.R

```diff
@@ -344,3 +344,20 @@ get_center_from_cpp_output <- function (res, what = "points") {
 
     return (as.data.frame (this))
 }
+
+
+#' Set encoding to UTF-8
+#'
+#' @param x a data.frame or a list.
+#'
+#' @return `x` with all the columns or items of type character with UTF-8 encoding set.
+#' @noRd
+setenc_utf8 <- function(x) {
+    char_cols <- vapply (x, is.character, FUN.VALUE = logical (1))
```

Comment:
```suggestion
    char_cols <- which (vapply (x, is.character, FUN.VALUE = logical (1)))
```
just to make everything more explicit (and technically more efficient).